src/api/localStorage.js

```diff
@@ -0,0 +1,111 @@
+import Enumerable from '../../node_modules/linq';
+import faker from '../../node_modules/faker';
+export class LocalStorage {
+  constructor(_id) {
+    this._id = _id;
+  }
+
+  createUser() {
```

Comment:
I would expect a user object (sans `_id`) to be passed into `createUser` to be used for the `userEntry`

Reply:
In localStorage.js I am creating the user to be pushed into localStorage within createUser(), should I create the user in the CreateUser Class using it's constructor and pass that in when I call createUser() from the CreateUser component?

Reply:
> In localStorage.js I am creating the user to be pushed into localStorage within createUser()

No, you'll create the user in your components (probably the create user one). That's where the inputs for the user data are, so that's where you should build out the bulk of the user object. 

The `_id` property can be added there as well, but since that's not something the form itself is concerned with, it would also make sense for you to add the `_id` param in your LocalStorage class function `createUser()`, before stringifying and sending it to `localStorage`.

> should I create the user in the CreateUser Class using it's constructor...

Probably not. You'll want to use the same `LocalStorage` instance throughout your app, not every time you need to create a new user. Plus, [you're already creating an `_id` in `createUser`](https://github.com/frankielb/frankie-barrios-ux-aop/pull/22/files/1dc882215786c38a6c448f061e24f05c3659d669#diff-a09ed0d6e500df7a8a529a6e5048a611R11) as it is. Rather than assigning it to `this._id`, assign it to the `userEntry._id` a few lines down.

> ... and pass that in when I call createUser() from the CreateUser component?

Like I said in the first answer, you'll want to have the component create the bulk of the user object and pass it into `createUser`. You won't need to pass anything into `createUser` that was given back from the initial creation of the `LocalStorage` instance. The instance should just give you the ability to use the class functions, for the most part.

Reply:
Good deal, thanks for such a detailed response. Helps a lot. Cutting a lot of the 'fluff' out of a majority of my functions as well with this refactor.